test/utils/test.mjs

```diff
@@ -112,6 +117,7 @@ class Tester {
 		const tester = snapshotRuleTester(test, {
 			parserOptions: defaultParserOptions
 		});
+		verifySuggestionPropertyConsistency(this.rule, tests.invalid);
```

Comment:
This won't work because snapshot tester don't need set `errors`, should check here https://github.com/sindresorhus/eslint-plugin-unicorn/blob/bb0446884238745fd565b3ebc64e6c0de3f2c31b/test/utils/snapshot-rule-tester.mjs#L105

Reply:
Fixed.